packages/core/src/constants/relayer.ts

```diff
@@ -32,4 +32,4 @@ export const RELAYER_STORAGE_OPTIONS = {
 
 // FIXME: Always manually change this after each release (hardcoded from package.json)
 // ... this will be temporarily hardcoded until we refactor our build chain!
-export const RELAYER_SDK_VERSION = "2.0.0-beta.102";
+export const RELAYER_SDK_VERSION = "2.0.0-rc.0";
```

Comment:
Again with the in-source version 😄 I think this needs to be either the canary version you published or we do rc.1 rather than merging a canary version to v2 after we already started RCs

Reply:
Yeh, my thinking here was basically that - update this when we release rc / prod